airflow/cli/commands/connection_command.py

```diff
@@ -29,7 +29,7 @@
 from airflow.exceptions import AirflowNotFoundException
 from airflow.hooks.base import BaseHook
 from airflow.models import Connection
-from airflow.secrets.local_filesystem import _create_connection, load_connections_dict
+from airflow.secrets.local_filesystem import load_connections_dict
```

Comment:
What do you think to add this method as a static method to the Connection class - `Connection.from_dict`?

Reply:
Ahhh. You don't want to change the visibility of this method, but you are using a different public method. We should therefore still create the `Connection.from_dict` method but as a separate PR.

Reply:
Thanks for taking a look. I can add `Connection.from_dict`.

For the implementation, should I avoid any dependencies on `secrets`? The existing function `load_connections_dict` calls a few helper functions. For example, it calls `_parse_secret_file`, which calls `_parse_[json,yaml,env]_file`.

I can either:
- move all these helper functions outside of `secrets/local_filesystem.py`
- duplicate the same functionality in the cli module
- add a method in the Connection class that still imports from secrets (but doesn't eliminate the dependency)

What do you have in mind?

Reply:
Yes. We should avoid dependency on secrets. We already have a lot of recursive imports between various connection modules, secret and others, so creating new modules will cure those situations a bit.

> move all these helper functions outside of secrets/local_filesystem.py

I think it is the best solution.

Reply:
You can do it in this PR.

Reply:
This change is now implemented.